src/js/components/stock-movement-wizard/inboundV2/InboundV2.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useMemo } from 'react';
+
+import Button from 'components/form-elements/Button';
+import InboundHeader from 'components/stock-movement-wizard/inboundV2/InboundHeader';
+import InboundV2AddItems
+  from 'components/stock-movement-wizard/inboundV2/sections/InboundV2AddItems';
+import InboundV2Create from 'components/stock-movement-wizard/inboundV2/sections/InboundV2Create';
+import InboundV2Send from 'components/stock-movement-wizard/inboundV2/sections/InboundV2Send';
+import WizardStepsV2 from 'components/wizard/v2/WizardStepsV2';
+import inboundV2Step from 'consts/InboundV2Step';
+import mockInboundV2AdditionalTitle from 'consts/MockInboundV2AdditionalTitle';
+import mockInboundV2Title from 'consts/MockInboundV2Title';
+import useTranslate from 'hooks/useTranslate';
+import useTranslation from 'hooks/useTranslation';
+import useWizard from 'hooks/useWizard';
+import PageWrapper from 'wrappers/PageWrapper';
+
+const InboundV2 = () => {
+  useTranslation('stockMovement');
+  const translate = useTranslate();
+
+  const steps = useMemo(() => [
+    {
+      key: inboundV2Step.CREATE,
+      title: translate('react.stockMovement.create.label', 'Create'),
+      Component: () => (<InboundV2Create />),
+    },
+    {
+      key: inboundV2Step.ADD_ITEMS,
+      title: translate('react.stockMovement.addItems.label', 'Add Items'),
+      Component: () => (<InboundV2AddItems />),
+    },
+    {
+      key: inboundV2Step.SEND,
+      title: translate('react.stockMovement.send.label', 'Send'),
+      Component: () => (<InboundV2Send />),
+    },
+  ], [translate]);
```

Comment:
translate is a function, I think we should use simple data as a dependency (in this case you can use language)